romana-install/roles/romana/install-agent/files/agent_proxy_add_policy.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
+# Agent Proxy for Adding Policies.
+#
```

Comment:
It would have been nice to see a few more comments in these hook scripts: Who calls it, when are they called, what are they doing, etc.